python/xorq/cli.py

```diff
@@ -100,13 +130,26 @@ def main():
         "--target-dir", default="build", help="Directory for all generated artifacts"
     )
 
-    # Parse the arguments
+    # Create parser for the "run" command
+    run_parser = subparsers.add_parser(
+        "run", help="Run a build from a builds directory"
+    )
+    run_parser.add_argument("builds_dir", help="Path to the builds directory")
```

Comment:
this argument should have the same name as the `build` command

i think its probably better to use `builds_dir` and change `build` command to use `--builds-dir`